source/qml/Library/RendererQml/AdaptiveCardQmlRenderer.cpp

```diff
@@ -603,34 +603,89 @@ namespace RendererQml
         //TODO: Add hover effect
         backgroundTag->Property("color", context->GetRGBColor(context->GetConfig()->GetContainerStyles().defaultPalette.backgroundColor));
 
+		backgroundTag->Property("border.color", Formatter() << inputId << "_contentItem" << ".activeFocus? 'black' : 'grey'");
 		backgroundTag->Property("layer.enabled", Formatter() << inputId + "_contentItem" << ".activeFocus ? true : false");
 		backgroundTag->Property("layer.effect", glowTag->ToString());
 
 		auto contentItemTag = std::make_shared<QmlTag>("TextField");
 		contentItemTag->Property("id", inputId + "_contentItem");
-		contentItemTag->Property("padding", "10");
 		contentItemTag->Property("font.pixelSize", std::to_string(context->GetConfig()->GetFontSize(AdaptiveSharedNamespace::FontType::Default, AdaptiveSharedNamespace::TextSize::Default)));
+		contentItemTag->Property("anchors.left", "parent.left");
 		contentItemTag->Property("selectByMouse", "true");
 		contentItemTag->Property("selectedTextColor", "'white'");
 		contentItemTag->Property("readOnly", Formatter() << "!" << inputId << ".editable");
 		contentItemTag->Property("validator", Formatter() << inputId << ".validator");
 		contentItemTag->Property("inputMethodHints", "Qt.ImhFormattedNumbersOnly");
-		contentItemTag->Property("text", Formatter() << inputId << ".textFromValue(" << inputId << ".value, " << inputId << ".locale)");
+		contentItemTag->Property("text", Formatter() << inputId << ".value");
```

Comment:
Why are we removing the function here?

Reply:
It is adding commas for numbers in thousands and more than that